src/http/input/body/N3PatchBodyParser.ts

```diff
@@ -0,0 +1,136 @@
+import type { DefaultGraph, NamedNode, Quad, Quad_Subject } from '@rdfjs/types';
+import { DataFactory, Parser, Store } from 'n3';
+import type { Term } from 'rdf-js';
+import { getBlankNodes, getTerms, getVariables, mapTerms } from 'rdf-terms';
+import { TEXT_N3 } from '../../../util/ContentTypes';
+import { BadRequestHttpError } from '../../../util/errors/BadRequestHttpError';
+import { createErrorMessage } from '../../../util/errors/ErrorUtil';
+import { UnprocessableEntityHttpError } from '../../../util/errors/UnprocessableEntityHttpError';
+import { UnsupportedMediaTypeHttpError } from '../../../util/errors/UnsupportedMediaTypeHttpError';
+import { guardedStreamFrom, readableToString } from '../../../util/StreamUtil';
+import { RDF, SOLID } from '../../../util/Vocabularies';
+import type { N3Patch } from '../../representation/N3Patch';
+import type { BodyParserArgs } from './BodyParser';
+import { BodyParser } from './BodyParser';
+
+/**
+ * Parses an N3 Patch document and makes sure it conforms to the specification requirements.
+ * Requirements can be found at Solid, §5.3.1: https://solid.github.io/specification/protocol#n3-patch
```

Comment:
```suggestion
 * Requirements can be found at Solid Protocol, §5.3.1: https://solid.github.io/specification/protocol#n3-patch
```
(or just Protocol; throughout)

Reply:
`Solid, §5.3.1` in front of a quote from the spec is how it is also currently done in all other classes. But here it does indeed make more sense to phrase it a bit differently since it's just a direct link to the spec.